src/Avalonia.Animation/Keyframes/IKeyFrames.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Avalonia.Animation.Keyframes
+{
+    /// <summary>
+    /// Interface for Keyframe group object
+    /// </summary>
+    public interface IKeyFrames
```

Comment:
I'm interested to know why someone would want to provide a different implementation of `IKeyFrames`?